libyul/YulControlFlowGraphExporter.cpp

```diff
@@ -128,12 +129,25 @@ Json YulControlFlowGraphExporter::exportBlock(SSACFG const& _cfg, SSACFG::BlockI
 	return blocksJson;
 }
 
-Json YulControlFlowGraphExporter::toJson(SSACFG const& _cfg, SSACFG::BlockId _blockId)
+Json YulControlFlowGraphExporter::toJson(SSACFG const& _cfg, SSACFG::BlockId _blockId, SSACFGLiveness const* _liveness)
 {
+	auto const valueToString = [&](SSACFG::ValueId const& valueId) { return varToString(_cfg, valueId); };
```

Comment:
I guess we could generalize the [SSACFGPrinter](https://github.com/ethereum/solidity/blob/6b000bded234f9749d5058772f1860c6205cd8dc/libyul/backends/evm/SSAControlFlowGraph.cpp#L42) and reuse it in the exporter, so the exporter could be an instance that implements its own `writeBlock` and `writeFunction` for different output formats.

Reply:
Yeah, there's a lot of duplication here :)

Reply:
I will address that in a follow-up PR :)